src/pages/api/alerts.ts

```diff
@@ -0,0 +1,54 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+
+import { Alert, AlertJsonResponse } from '../../lib/types'
+import { getLogger } from '../../logging/log-util'
+
+const logger = getLogger('get-alerts')
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Alert[] | string>
+) {
+  if (!process.env.ALERT_JSON_URI) {
```

Comment:
I'm suggesting to log a warning message and returning a 404 http status code.

Reply:
A REST collection endpoint shouldn't return 404, in my opinion. It should return an empty collection.